README.md

```diff
@@ -69,8 +69,8 @@ git push
 
 ### Versioning the registered package
 
-1. Update the major or minor version numbers in the corresponding field at the top of `Project.toml`
-2. Add `@JuliaRegistrator register` in a comment in the commit you wish to use for the release
+1. Start a new branch and update the major or minor version numbers in the corresponding field at the top of `Project.toml`
+2. After pushing the changes and starting a new PR, add `@JuliaRegistrator register` in a comment in the commit you wish to use for the release (not a comment on the PR, but the actual commit)
```

Comment:
This should be something like "after merging the changes". A PR is not necessary.